samples/dnn/dasiamrpn_tracker.py

```diff
@@ -0,0 +1,281 @@
+        target_size = target_size / scale_z
+        lr = penalty[best_pscore_id] * score[best_pscore_id] * self.lr
+        res_x = target[0] + self.target_pos[0]
+        res_y = target[1] + self.target_pos[1]
+        res_w = target_size[0] * (1 - lr) + target[2] * lr
+        res_h = target_size[1] * (1 - lr) + target[3] * lr
+        self.target_pos = np.array([res_x, res_y])
+        self.target_sz = np.array([res_w, res_h])
+        return score[best_pscore_id]
+
+    def __softmax(self, x):
+        x_max = x.max(0)
+        e_x = np.exp(x - x_max)
+        y = e_x / e_x.sum(axis = 0)
+        return y
+
+    #evaluations with cropped image
+    def __get_subwindow_tracking(self, im, model_size, original_sz):
+        im_sz = im.shape
+        c = (original_sz + 1) / 2
+        context_xmin = round(self.target_pos[0] - c)
+        context_xmax = context_xmin + original_sz - 1
+        context_ymin = round(self.target_pos[1] - c)
+        context_ymax = context_ymin + original_sz - 1
+        left_pad = int(max(0., -context_xmin))
+        top_pad = int(max(0., -context_ymin))
+        right_pad = int(max(0., context_xmax - im_sz[1] + 1))
+        bottom_pad = int(max(0., context_ymax - im_sz[0] + 1))
+        context_xmin = context_xmin + left_pad
+        context_xmax = context_xmax + left_pad
+        context_ymin = context_ymin + top_pad
+        context_ymax = context_ymax + top_pad
+        r, c, k = im.shape
+
+        if any([top_pad, bottom_pad, left_pad, right_pad]):
+            te_im = np.zeros((r + top_pad + bottom_pad, c + left_pad + right_pad, k), np.uint8)
+            te_im[top_pad:top_pad + r, left_pad:left_pad + c, :] = im
+            if top_pad:
+                te_im[0:top_pad, left_pad:left_pad + c, :] = self.avg_chans
+            if bottom_pad:
+                te_im[r + top_pad:, left_pad:left_pad + c, :] = self.avg_chans
+            if left_pad:
+                te_im[:, 0:left_pad, :] = self.avg_chans
+            if right_pad:
+                te_im[:, c + left_pad:, :] = self.avg_chans
+            im_patch_original = te_im[int(context_ymin):int(context_ymax + 1), int(context_xmin):int(context_xmax + 1), :]
+        else:
+            im_patch_original = im[int(context_ymin):int(context_ymax + 1), int(context_xmin):int(context_xmax + 1), :]
+
+        if not np.array_equal(model_size, original_sz):
+            im_patch = cv.resize(im_patch_original, (model_size, model_size))
+        else:
+            im_patch = im_patch_original
+
+        return im_patch.astype(float)
+
+#function for reading paths, bounding box drawing, showing results
+def main():
+    parser = argparse.ArgumentParser(description="Run tracker")
+    parser.add_argument("--net", type=str, help="Full path to onnx model of net")
+    parser.add_argument("--kernel_r1", type=str, help="Full path to onnx model of kernel_r1")
+    parser.add_argument("--kernel_cls1", type=str, help="Full path to onnx model of kernel_cls1")
```

Comment:
Please add `required=True` to argument or use default args. For example, `default=dasiamrpn_model.onnx`

Reply:
Done